tests/vmi_configuration_test.go

```diff
@@ -2373,17 +2376,20 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 
 			By("creating a disk image")
 			var nodeName string
-			tmpHostDiskDir := storage.RandTmpDir()
+			tmpHostDiskDir := storage.RandHostDiskDir()
 			tmpHostDiskPath := filepath.Join(tmpHostDiskDir, fmt.Sprintf("disk-%s.img", uuid.NewString()))
 
-			pod := storage.CreateDiskOnHost(tmpHostDiskPath)
+			pod := storage.CreateHostDisk(tmpHostDiskPath)
 			pod, err := virtClient.CoreV1().Pods(testsuite.NamespacePrivileged).Create(context.Background(), pod, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 			Eventually(ThisPod(pod), 30*time.Second, 1*time.Second).Should(BeInPhase(k8sv1.PodSucceeded))
 			pod, err = ThisPod(pod)()
 			Expect(err).NotTo(HaveOccurred())
 			nodeName = pod.Spec.NodeName
-			defer storage.RemoveHostDiskImage(tmpHostDiskDir, nodeName)
+			defer func() {
+				err := storage.RemoveHostDisk(tmpHostDiskDir, nodeName)
```

Comment:
Same
`Expect(...).To(Succeed())`

But this test should be fixed as a follow up as defer is usually not a good fit.